django/conf/project_template/project_name/settings.py

```diff
@@ -82,6 +82,22 @@
 }
 
 
+# Password validation
+# https://docs.djangoproject.com/en/{{ docs_version }}/ref/settings/#auth-password-validators
+
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
```

Comment:
Why is NAME in capitals? 

For comparison, DATABASES and CACHES use lower-case 'default' and then capitals, LOGGING uses lower-case only.